charts/backstage/templates/sonataflows.yaml

```
@@ -0,0 +1,133 @@
+{{- if and (default false .Values.orchestrator.enabled) (default false .Values.orchestrator.serverlessLogicOperator.enabled) }}
+{{- $sonataflowplatformExists := lookup "sonataflow.org/v1alpha08" "SonataFlowPlatform" .Release.Namespace "sonataflow-platform" }}
+{{- if $sonataflowplatformExists }}
```

```suggestion
{{- if and .Release.IsInstall $sonataflowplatformExists }}
```

Otherwise, this prevents upgrading an existing Release (so same RHDH instance), which is an issue, I think:
```
Error: UPGRADE FAILED: 
execution error at (backstage/templates/sonataflows.yaml:4:4): 
Cannot create multiple sonataflowplatform in the same namespace, one already exists.
```